spec/javascripts/components/layout-super-navigation-header-spec.js

```diff
@@ -374,6 +380,17 @@ describe('The super header navigation', function () {
 
       expect(document.activeElement).toEqual($lastNavLink)
     })
+
+    it('when the last search menu link is focussed and the tab key is pressed the search menu is closed', function () {
```

Comment:
I think we can make this a bit more descriptive now it is checking for more than the last link element, for example:

"when the last tabbable element in the search menu is focussed and the tab key is pressed the search menu is closed"